pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/MailService.java

```diff
@@ -20,53 +22,79 @@
 import java.io.FileReader;
 import java.io.IOException;
 import java.io.StringWriter;
-import java.io.Writer;
+import java.util.Map;
 
 public class MailService {
 	private static Logger logger = LoggerFactory.getLogger(MailService.class);
 	private static MustacheFactory mf = new DefaultMustacheFactory();
 
-	private static Mustache accessEmail = null;
-	
-	private void loadTemplates() {
-		try(FileReader reader = new FileReader(JAXRSConfiguration.userActivationTemplatePath);){
-			accessEmail = mf.compile(reader, "accessEmail");
+	private Mustache loadTemplates(String templateFile) throws IOException {
+		try(FileReader reader = new FileReader(JAXRSConfiguration.emailTemplatePath + templateFile)){
+			return mf.compile(reader, templateFile);
 		} catch (FileNotFoundException e) {
-			e.printStackTrace();
+			logger.error(e.getMessage(), e);
+			throw e;
 		} catch (IOException e) {
-			e.printStackTrace();
+			logger.error(e.getMessage(), e);
+			throw e;
```

Comment:
There is no need to throw this exception, nothing upstream in the call stack, nor the user, can do anything about this issue. The exception in the log is all that admins need.

Additionally, the action which triggered this email should not be prevented if the email did not send. 

Same for the catch block immediately before this one